nixpkgs_review/nix/evalAttrs.nix

```diff
@@ -1,8 +1,10 @@
-attr-json:
+{ attr-json
+, allowUnfree ? false
+}:
 
 with builtins;
 let
-  pkgs = import <nixpkgs> { config = { checkMeta = true; }; };
+  pkgs = import <nixpkgs> { config = { checkMeta = true; inherit allowUnfree; }; };
```

Comment:
I actually don't want this to be a switch. Every user should get the same result when running nixpkgs on pr. If there is problematic propritary software in nixpkgs, we should rather remove it than pretending its not there. Just enable it by default I would say.

Reply:
I think that's fair. I'll revise this